service/service.py

```diff
@@ -143,6 +143,28 @@ def create_promotions():
         jsonify(message), status.HTTP_201_CREATED, {"Location": location_url}
     )
 
+######################################################################
+# ADD A NEW PRODUCT
+######################################################################
+@app.route("/products", methods=["POST"])
+def create_product():
+    """
+    Creates a Product
+    This endpoint will create a Product based the id in the body that is posted
+    """
+    app.logger.info("Request to create a product")
+    check_content_type("application/json")
+    product = Product()
+    product.deserialize(request.get_json())
+    product.create()
+    message = product.serialize()
+    location_url = url_for("get_promotions", promotion_id=product.id, _external=True)
```

Comment:
umm... Do we'll need a Read function too?
```suggestion
    location_url = url_for("get_products", product_id=product.id, _external=True)
```